packages/node_modules/@ciscospark/redux-module-activity/src/actions.test.js

```diff
@@ -168,7 +168,7 @@ describe('redux-module-activity actions', () => {
     it('saves a failed share activity', () => {
       const store = mockStore({});
       const activity = initialState
-        .set('text', 'I\'m walking here!')
+        .set('text', '**bold** yo!')
```

Comment:
Instead of using the same text for every test, should we switch it up a bit so we can also test different types of markup to get better coverage?